src/app/categories/[categoryId]/pairwise-ranking/comment/page.tsx

```diff
@@ -293,12 +293,10 @@ const CategoryRankingComment = () => {
 			});
 
 			router.push(
-				`${Routes.Categories}/${category?.data.collection?.id}/pairwise-ranking/done`,
+				`${Routes.Categories}/${category?.data.collection?.id}/pairwise-ranking/vote-submitted`,
 			);
 		} catch (e) {
 			console.error('error on sending tx:', e);
-		} finally {
-			setAttestUnderway(false);
```

Comment:
Removing this, if for any reason, the attestation doesn't go through and faces an error, users are going to be stuck with the `Submitting vote` modal thinking that it's still in progress under the hood.